po/POTFILES.in

```diff
@@ -415,6 +415,7 @@ src/tests/eina/eina_test_quaternion.c
 src/tests/eina/eina_test_strbuf.c
 src/tests/eina/eina_test_lalloc.c
 src/tests/eina/eina_test_bezier.c
+src/tests/eina/eina_test_thread.c
```

Comment:
hey @JPTIZ you know if we have to add things to `po`?

Reply:
@cauebs why you added the test case to `po`?

Reply:
To be quite honest, I just mirrored the other test cases.

Reply:
> hey @JPTIZ you know if we have to add things to `po`?

Well, we never added a new test file before.